tools/WinObjC.Packaging/WinObjC.Packager.targets

```diff
@@ -0,0 +1,70 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGER TARGETS
+  Description: The target definitions specific to a Nugetizing project for WinObjC
+  For Use By: A NuProj that is doing the packaging of one or more projects
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_CoreCompileOverrideDependsOn Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">Pack</_CoreCompileOverrideDependsOn>
+  </PropertyGroup>
+
+  <Target Name="CoreCompile" DependsOnTargets="$(_CoreCompileOverrideDependsOn)" />
+
+  <Target Name="AddSingleConfigBuildOutput"  Returns="@(_PackageContentsForPlat)">
+    <MSBuild Projects="$(SolutionPath)"
+       Targets="GetSolutionConfigurationContents"
+       Properties="Configuration=$(Configuration);Platform=$(Platform)">
+      <Output TaskParameter="TargetOutputs" ItemName="_SolutionConfigurationContents" />
+    </MSBuild>
+
+    <Message Text="AddSingleConfigBuildOutput Platform: $(Platform)" Importance="High"/>
+
```

Comment:
I used this for debugging. Maybe turn this into a more human friendly version with a lower pri if we feel its still nice to keep around.